addons/knowledge/tests/test_knowledge_article_permissions.py

```diff
@@ -273,12 +273,19 @@ def test_remove_member_inherited_rights(self):
                            {self.partner_employee_manager: 'write'})
 
         # remove partner employee manager that has rights based on inheritance
+        writable_child = self.article_write_contents_children[0].with_env(self.env)
+        self.assertTrue(
```

Comment:
Should use "assertIn" and "assertNotIn" for those, as it makes reading a bit easier.
(I let you update all occurrences, will not repeat the comment every time ^^)